pkmn.py

```diff
@@ -293,8 +263,9 @@ def opponent_choose_action(x, bookkeeper, action_space):
             # with values greater than 32.
             pvec[i] -= pvec_max - 32
     pvec = np.exp(pvec)
-    for i in range(len(OPPONENT_POSSIBLE_ACTIONS)):
-        pvec[i] *= OPPONENT_POSSIBLE_ACTIONS[i] in action_space
+    if (len(sys.argv) == 2 or debug):
```

Comment:
Maybe set debug = True if len(sys.argv) == 2, just to make a bit more robust.

Reply:
Done